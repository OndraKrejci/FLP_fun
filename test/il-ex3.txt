S,A,B
a,b
S
S->aA
S->bB
S->b
A->Baa
A->ba
B->bAAb
B->ab