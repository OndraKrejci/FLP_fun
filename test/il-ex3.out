a',b',S,A,B,<aa>,<AAb>,<Ab>
a,b
S
a'->a
b'->b
S->a'A
S->b'B
S->b
A->b'a'
B->a'b'
A->B<aa>
<aa>->a'a'
B->b'<AAb>
<AAb>->A<Ab>
<Ab>->Ab'